x-element.js

```diff
@@ -1756,10 +1756,6 @@ class TemplateEngine {
     }
   }
 
-  static #createReference() {
-    return Object.create(null);
```

Comment:
We’re simply trading `Object.create(null)` for `Symbol()` in this code. To me, calling `Symbol()` (note, not `Symbol.for()`) has better semantics — it’s pretty clear that we just want a guaranteed-unique reference there.